### DISCUSSION TITLE
What's the difference between Request and Force for notification settings?

### DISCUSSION THREAD

**Discussion by korboybeats** [runelite/runelite#18617] [Q&A]
Both settings force Runelite to the front.

**Comment by Felanbird** [comment]
edit: Sometimes the `Request` can be processed by your Operating System and force the client to the front, the `Force` option is our intended way to force the client, as mentioned by the old part of my comment below, `Request` and `Force` do 2 clearly different things for me.

<br>
_only_ `Force` forces the client to the front, you might be changing the global setting, while having a per-setting setup 

per-setting
![image](https://github.com/user-attachments/assets/21fcf517-c909-4d78-b1f5-d8b99081ad20)
![image](https://github.com/user-attachments/assets/a52a82c7-8d15-4ddb-9744-6f54bb552821)
![image](https://github.com/user-attachments/assets/b51e1d39-068c-469c-bc7b-416ccef29eaf)

global
![image](https://github.com/user-attachments/assets/2404513e-6650-4dee-90dc-15b142f70d11)
